src/Documentation/sidebar.json

```diff
@@ -41,8 +41,7 @@
     "labels": {
       "data-and-model-files-versioning.md": "Data & Model Files Versioning",
       "share-data-and-model-files.md": "Share Data & Model Files",
-      "multiple-data-scientists-on-a-single-machine.md":
-        "Shared Development Machine"
+      "multiple-data-scientists-on-a-single-machine.md": "Shared Development Machine"
```

Comment:
I believe this line now is > 80 symbols. Please, don't forget to run prettier for JS and MD files.